homeassistant/components/uvc/camera.py

```diff
@@ -114,32 +137,30 @@ def brand(self):
     @property
     def model(self):
         """Return the model of this camera."""
-        caminfo = self._nvr.get_camera(self._uuid)
-        return caminfo['model']
+        return self._caminfo['model']
 
     def _login(self):
         """Login to the camera."""
         from uvcclient import camera as uvc_camera
 
-        caminfo = self._nvr.get_camera(self._uuid)
         if self._connect_addr:
             addrs = [self._connect_addr]
         else:
-            addrs = [caminfo['host'], caminfo['internalHost']]
+            addrs = [self._caminfo['host'], self._caminfo['internalHost']]
 
         if self._nvr.server_version >= (3, 2, 0):
             client_cls = uvc_camera.UVCCameraClientV320
         else:
             client_cls = uvc_camera.UVCCameraClient
 
-        if caminfo['username'] is None:
-            caminfo['username'] = 'ubnt'
+        if self._caminfo['username'] is None:
+            self._caminfo['username'] = 'ubnt'
```

Comment:
This can't work. `None['username']` is not a thing.

Reply:
I think it's saying that if the username is not set, to use the default 'ubnt', right? or am I looking at this wrong?

Reply:
Oh snap, you're right. My bad !